### ISSUE TITLE
use turtlebot model to replace car

### ISSUE THREAD

**Issue by jianghe01** [Microsoft/AirSim#1576]
Hi, i want to use turtlebot model to replace the car model to run airsim.
first i created a turtlebot fbx model by maya and import it to unreal. i set the physics bodies of bones to kinematic type to prevent them dividing.
secondly i created tire data and animation blueprint.
then i created a wheeledvehicle blueprint class inherited from carpawn. i set the class as you say in the PawnPaths settings and change the PawnPaths settings.
i also modified the input of project settings.

now it can render the turtlebot model to replace the car model. but i cannot move it by keyboard or program. can anyone help me to do it correctly. thanks a lot

**Comment by RVBldr** [comment]
@jianghe01 This is an Unreal question, so the best way forward is to review Unreal documentation on vehicle creation, as well as the many YouTube videos on creating actors for Unreal. That should help you with the control mappings.

**Comment by shaoxiang** [comment]
@jianghe01 Have you solved the problem? I also want to use robots in airsim.

**Comment by jianghe01** [comment]
No, I have not solved this problem. Sorry I cannot give you any more advice because I’m not working on the project now.

发件人: shaoxiang [mailto:notifications@github.com]
发送时间: 2020年6月1日 星期一 13:12
收件人: microsoft/AirSim
抄送: Jiang,He; Mention
主题: Re: [microsoft/AirSim] use turtlebot model to replace car (#1576)


@jianghe01<https://mailshield.baidu.com/check?q=RL%2fJUytPs%2bTh67neRWiYgruHgH8AROnlYEzVG3qd8m0%3d> Have you solved the problem? I also want to use robots in airsim.

**Comment by shaoxiang** [comment]
@jianghe01  It's sad to hear that this problem can't be solved in airsim...

**Comment by jonyMarino** [comment]
related to #1506

**Comment by zhoudacheng6** [comment]
Have someone solved the problem? I also want to use robots in airsim.

**Comment by jonyMarino** [comment]
Take a look at the urdfAirSim repository for examples
